drop-storage/src/lib.rs

```diff
@@ -232,8 +233,10 @@ impl Storage {
         let mut conn = self.conn.acquire().await?;
 
         sqlx::query!(
-            "INSERT INTO outgoing_path_pending_states (path_id) VALUES ((SELECT id FROM \
```

Comment:
Actually, it's not a simple query rewrite, it was wrong before because it made a subquery that could return 0 rows, and with INSERT it treated it as all NULL value single row. Whereas I rewrote it to use the INSERT SELECT query which works by inserting exactly the rows returned by SELECT.
It's a single statement now and not INSERT with subquery anymore

Reply:
True, will add this too

Reply:
Done